### DISCUSSION TITLE
Source code on free and open source alternative to GitHub

### DISCUSSION THREAD

**Discussion by linux-lukas** [organicmaps/organicmaps#2888] [Q&A]
On the one hand I can understand why GitHub as a platform for providing source code, on the other hand there have been several developments over the years now that are conceivable . 2018 the purchase of Microsoft for example.

The latest development, CO-Pilot, I think is something that could hurt great open source projects like this in terms of licenses.

I think it would be good for the project to make the source code of the Organicmaps project available via a free and open source alternative, such as the Gitea-based source code delivery platform Codeberg.

**Comment by biodranik** [comment]
The power is in the community that helps to improve the project. What are your concerns? And why do you think a smaller platform run by someone else is better?

**Reply by linux-lukas** [reply]
Well, my concerns are that I don't think it can be good for open source projects if source code delivery platforms are owned by profit-oriented, mostly big tech corporations. 

Alternatively, a mirror of the repository could be done.

GitHub is run by a for-profit corporation - Microsoft. The Gitea-based Codeberg project is run by a non-profit, non-governmental organisation based in Berlin, Germany, Codeberg e. V..

**Reply by biodranik** [reply]
Above I described the value and benefits for open source projects from big platforms. Making a clone/copy is easy at any time, but it requires maintenance and time. To do it "just for fun" is not fun.

**Reply by linux-lukas** [reply]
Of course, I understand the points you mentioned and I can fully agree with them. Nevertheless, the point of a complete migration or double operation via mirror should perhaps be looked at.

**Reply by linux-lukas** [reply]
Btw: The Pixelfed maintainer has written on Mastodon that he is considering migrating the Pixelfed project away from GitHub to Codeberg: https://mastodon.social/@dansup/10869007146309141

Furthermore, Aral has written - also on Mastodon - for his GitHub projects as well as those of his Foundation, which he has migrated or is migrating to Codeberg together with another person, Small Technology: https://mastodon.ar.al/@aral/108651118032076133

@biodranik
